google/test_utils.go

```diff
@@ -51,3 +55,33 @@ func toBool(attribute string) (bool, error) {
 	}
 	return strconv.ParseBool(attribute)
 }
+
+func testAccDataSourceMatchesResourceCheck(dataSourceName string, resourceName string, attrsToCheck []string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		ds, ok := s.RootModule().Resources[dataSourceName]
+		if !ok {
+			return fmt.Errorf("root module has no resource called %s", dataSourceName)
```

Comment:
Can you reword the error here and on line 68? I think this should say "module has no data source" instead of "resource" and the second error should be explicit that it can't find the "Resource"